### PR TITLE
Muhannad React week4

### PR DESCRIPTION
**Pull request by Muhannad1989**

### CODE REVIEW

File: homework-submission/react-hyf/react-hyf-client/package.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+  "name": "react-hyf-client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "bootstrap": "^4.3.1",
+    "react": "^16.8.6",
+    "react-dom": "^16.8.6",
+    "react-scripts": "3.0.1"
```
Comment (Meno-101) [inline homework-submission/react-hyf/react-hyf-client/package.json]: When you install a package make sure you have it in your `package.json` file, when I ran your application first error I found:

Screen Shot 2019-07-17 at 06 46 42

File: homework-submission/react-hyf/react-hyf-client/src/components/navbar/navbar.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './navbar.css';
+
+const NavBar = () => {
+  return (
+    <div className="menu">
+      <ul class="">
```
Comment (Meno-101) [inline homework-submission/react-hyf/react-hyf-client/src/components/navbar/navbar.js]: A super easy fix:)

Screen Shot 2019-07-17 at 06 51 51

Comment (Muhannad1989) [reply homework-submission/react-hyf/react-hyf-client/src/components/navbar/navbar.js]: You are right , it is className 😅

**Comment by Muhannad1989** [comment]
Thank you @meno -101 for all things you did for me. I appreciate that 😀

File: homework-submission/react-hyf/react-hyf-client/package.json (hunk 1 above)
Comment (Muhannad1989) [reply homework-submission/react-hyf/react-hyf-client/package.json]: By me it's still working 🤔, might i've removed it accidentally from the package.json after i totally finished.

**Pull request closed without merging**